src/clj/rems/api/permissions.clj

```diff
@@ -0,0 +1,22 @@
+(ns rems.api.permissions
+  (:require [compojure.api.sweet :refer :all]
+            [rems.api.schema :refer :all]
+            [rems.api.util] ; required for route :roles
+            [rems.db.entitlements :as entitlements]
+            [ring.util.http-response :refer :all]
+            [schema.core :as s]))
+
+(s/defschema GetPermissionsResponse
+  Ga4ghVisa)
+
+(def permissions-api
+  (context "/permissions" []
+    :tags ["permissions"]
+
+    (GET "/:user" []
+      :summary "Returns user's permissions in ga4gh visa format. See https://github.com/ga4gh-duri/ga4gh-duri.github.io/"
+      :roles #{:logged-in}
+      :path-params [user :- (describe s/Str "return permissions for this user (optional), ignored if the user doesn't have appropriate privileges")]
```

Comment:
"ignored if the user doesn't have appropriate privileges" sounds like it would ignore the parameter i.e. return permissions for all users

also, the user parameter doesn't look optional to me

Reply:
What would be better parameter instead of user?